$MirOS: ports/www/lynx/patches/patch-lynx_help_Lynx_users_guide_html,v 1.6 2010/12/23 20:25:31 tg Exp $

	site policy; some fix spelling

--- lynx_help/Lynx_users_guide.html.orig	Mon Feb 13 02:45:53 2012
+++ lynx_help/Lynx_users_guide.html	Sat Mar  3 18:20:22 2012
@@ -654,7 +654,7 @@ directly to the net.bio.net WAIS server.
   <p>The Lynx <em>Options Menu</em> may be accessed by pressing the
   '<em>o</em>' key. It allows you to change options at runtime, if
   you need to. Most changes are read from &amp; saved to your
-  .lynxrc file; those which are not are marked (!) in the
+  .etc/lynxrc file; those which are not are marked (!) in the
   form-based menu (as below). Many other options are stored in the
   <a href="#lynx.cfg">lynx.cfg</a> file.</p>
 
@@ -711,7 +711,7 @@ directly to the net.bio.net WAIS server.
   Personal mail address            : __________________________________________
   Personal mail name               : __________________________________________
   Password for anonymous ftp       : __________________________________________
-  Preferred media type (!)         : [Accept lynx's internal types]
+  Preferred media type (!)         : [Accept lynx' internal types]
   Preferred encoding (!)           : [All_____]
   Preferred document character set : _________________________________
   Preferred document language      : _________________________________
@@ -751,7 +751,7 @@ directly to the net.bio.net WAIS server.
 
      (E)ditor                     : emacs
      (D)ISPLAY variable           : aixtest.cc.ukans.edu:0.0
-     mu(L)ti-bookmarks: OFF       B)ookmark file: lynx_bookmarks.html
+     mu(L)ti-bookmarks: OFF       B)ookmark file: ./.etc/bookmark.htm
      (F)TP sort criteria          : By Filename
      (P)ersonal mail address      : montulli@netscape.com
      (S)earching type             : CASE INSENSITIVE
@@ -792,7 +792,7 @@ prompt.
 
   <p>When you are done changing options use the '<em>r</em>'
   command to return to Lynx or the '<em>&gt;</em>' command to save
-  the options to a <em>.lynxrc</em> file and return to Lynx.</p>
+  the options to a <em>.etc/lynxrc</em> file and return to Lynx.</p>
 
   <p>The following table describes the options available on the
   <em>Options Menu</em>:</p>
@@ -821,7 +821,7 @@ prompt.
     <dt>Bad HTML messages</dt>
 
     <dd>
-      Suppress or redirect Lynx's messages about "Bad HTML":
+      Suppress or redirect Lynx' messages about "Bad HTML":
 
       <dl>
         <dt>Ignore</dt>
@@ -1163,7 +1163,7 @@ prompt.
       the Accept.
 
       <dl>
-        <dt>Accept lynx's internal types</dt>
+        <dt>Accept lynx' internal types</dt>
 
         <dd>list only the types that are compiled into lynx.</dd>
 
@@ -1184,7 +1184,7 @@ prompt.
 
         <dt>Accept all types</dt>
 
-        <dd>adds the types that are in lynx's built-in tables for
+        <dd>adds the types that are in lynx' built-in tables for
         external programs that may be used to present a
         document.</dd>
       </dl>
@@ -1226,7 +1226,7 @@ prompt.
     with a message. If set to OFF or NEVER, color mode will be
     turned off.<br>
     ALWAYS and NEVER are not offered in anonymous accounts. If
-    saved to a <em>.lynxrc</em> file in non-anonymous accounts,
+    saved to a <em>.etc/lynxrc</em> file in non-anonymous accounts,
     ALWAYS will cause Lynx to set color mode on at startup if
     supported. If Lynx is built with the slang library, this is
     equivalent to having included the <em>-color</em> command line
@@ -1241,7 +1241,7 @@ prompt.
     switch, but (when the slang library is used) can be overridden
     with the <em>-color</em> switch.<br>
     If the setting is OFF or ON when the current options are saved
-    to a <em>.lynxrc</em> file, the default startup behavior is
+    to a <em>.etc/lynxrc</em> file, the default startup behavior is
     retained, such that color mode will be turned on at startup
     only if the terminal info indicates that you have a
     color-capable terminal, or (when the slang library is used) if
@@ -1371,7 +1371,7 @@ prompt.
 
     <dd><em id="noteUA">NOTE:</em> Some sites may regard
     misrepresenting the browser as fraudulent deception, or as
-    gaining unauthorized access, if it is used to circumvent
+    gaining unauthorised access, if it is used to circumvent
     blocking that was intentionally put in place. Some browser
     manufacturers may find the transmission of their product's name
     objectionable. If you change the User-Agent string, it is your
@@ -1462,7 +1462,7 @@ prompt.
   ownership. (Note to authors: if you want to assign the ownership
   to your document, you need to add into HEAD section a LINK
   element with appropriate value for REV attribute. Two values are
-  recognized: <em>owner</em> and <em>made</em> (these are case
+  recognised: <em>owner</em> and <em>made</em> (these are case
   insensitive). For example,</p>
   <pre>
 &lt;HEAD&gt;
@@ -2349,14 +2349,14 @@ with a no-cache directive.
   <p>You can also use the <em>Options Menu</em>, as outlined
   below:</p>
   <pre>
-     key  lynx.cfg       FM KM .lynxrc    variable in source
+     key  lynx.cfg       FM KM .etc/lynxrc variable in source
 
-       *  MAKE_LINKS_     Y  N       N    clickable_images
-       [  MAKE_PSEUDO_    Y  N       N    pseudo_inline_alts
-          VERBOSE_        Y  Y       Y    verbose_img
+       *  MAKE_LINKS_     Y  N       N     clickable_images
+       [  MAKE_PSEUDO_    Y  N       N     pseudo_inline_alts
+          VERBOSE_        Y  Y       Y     verbose_img
 
 FM = Form-based Menu ; KM = Key-based Menu ;
-in  .lynxrc ,  VERBOSE_IMAGES  is called `verbose_images':
+in  .etc/lynxrc ,  VERBOSE_IMAGES  is called `verbose_images':
 the other two cannot be saved between sessions.
 </pre>
 
@@ -2375,7 +2375,7 @@ the other two cannot be saved between se
   as arrays of cells aligned by columns and rows on the displayed
   page.</p>
 
-  <p>Lynx recognizes the TABLE element and all of its associated
+  <p>Lynx recognises the TABLE element and all of its associated
   elements as described in <a href=
   "http://www.faqs.org/rfcs/rfc1942.html">RFC 1942</a> and will
   process any ID attributes in the start tags for handling as
@@ -2438,7 +2438,7 @@ the other two cannot be saved between se
   make use of external scripts or programs. The normal Lynx
   distribution currently does not provide such scripts, but they
   can be written locally or downloaded from several sources. It is
-  suggested to use one of Lynx's facilities for invoking external
+  suggested to use one of Lynx' facilities for invoking external
   programs (see <kbd>DOWNLOADER</kbd>, <kbd>PRINTER</kbd>,
   <kbd>EXTERNAL</kbd>, <kbd>TRUSTED_LYNXCGI</kbd> in <a href=
   "#lynx.cfg">lynx.cfg</a> and <a href=
@@ -2484,7 +2484,7 @@ the other two cannot be saved between se
   of simultaneously displayed, independently scrolling windows.
   Such windows have been termed <em>frames</em>.</p>
 
-  <p>Lynx recognizes the Netscape and Microsoft Explorer FRAME,
+  <p>Lynx recognises the Netscape and Microsoft Explorer FRAME,
   FRAMESET, and NOFRAMES elements, but is not capable of windowing
   to create the intended positioning of <em>frames</em>. Instead,
   Lynx creates labeled links to the <em>frame</em> sources,
@@ -2514,7 +2514,7 @@ the other two cannot be saved between se
   analogous to a Windows toolbar. Such windows have been termed
   <em>banners</em>.</p>
 
-  <p>Lynx recognizes and processes all of the <a href=
+  <p>Lynx recognises and processes all of the <a href=
   "http://www.w3.org/MarkUp/html3/Contents.html">HTML 3.0</a> REL
   attribute tokens in LINK elements for creating a <em>banner</em>,
   and a number of others which have subsequently been proposed.
@@ -2537,7 +2537,7 @@ the other two cannot be saved between se
   value in the LINK, will invoke creation of a <em>banner</em> at
   the top of the first page, with the element's HREF as the link,
   and the token as the default link name. If a TITLE attribute is
-  included in the LINK, it's value will be used as the link name
+  included in the LINK, its value will be used as the link name
   instead of the default. <em>Bookmark</em> and <em>Banner</em> are
   intended to be accompanied by a TITLE attribute, which in effect
   makes the namespace for REL <em>banner</em> tokens infinite.</p>
@@ -2549,12 +2549,12 @@ the other two cannot be saved between se
   <em>STARTFILE</em> (i.e., derived from the configuration files or
   the WWW_HOME environment variable, <em>not</em> the command line
   <em>startfile</em> if one was used). However, if a
-  <em>-homepage=URL</em> was specified on the command line, it's
+  <em>-homepage=URL</em> was specified on the command line, its
   URL will be used as the HREF. For the special token
   <em>Index</em> without an HREF, Lynx will use the
   <em>DEFAULT_INDEX_FILE</em> derived from the configuration files,
   or if an <em>-index=URL</em> was specified on the command line,
-  it's URL will be used as the HREF.</p>
+  its URL will be used as the HREF.</p>
 
   <p>Lynx does not waste screen real estate maintaining the
   <em>banner</em> at the top of every page, but the Lynx TOOLBAR
@@ -2569,18 +2569,18 @@ the other two cannot be saved between se
   corner of the screen when the second or subsequent pages of the
   document are being displayed.</p>
 
-  <p>Lynx also recognizes the <a href=
+  <p>Lynx also recognises the <a href=
   "http://www.w3.org/MarkUp/html3/Contents.html">HTML 3.0</a>
   BANNER container element, and will create a <em>banner</em> based
   on its content if one has not already been created based on LINK
   elements. Lynx treats the Microsoft MARQUEE element as a synonym
-  for BANNER (i.e., presenting it's markup as a static
+  for BANNER (i.e., presenting its markup as a static
   <em>banner</em>, without any horizontal scrolling of its
   content). Lynx does not prefix the BANNER or MARQUEE content with
   a '<em>#</em>' because the content need not be only a series of
   links with brief, descriptive links names, but does add a
   '<em>#</em>' at the top, left-hand corner of the screen when the
-  content is not being displayed, to indicate it's accessibility
+  content is not being displayed, to indicate its accessibility
   via the TOOLBAR keystroke command.</p>
 
   <p>[<a href="#ToC-Banners">ToC</a>]</p>
@@ -2625,9 +2625,9 @@ of:
   new paragraph were indicated whether or not paragraphing markup
   is included in its content, with greater than normal left and
   right margins, and with the type of note indicated by an
-  emphasized label based on the value of its CLASS or ROLE
+  emphasised label based on the value of its CLASS or ROLE
   attribute. If no CLASS or ROLE attribute is included, the default
-  label <em>NOTE:</em> will be used. Lynx recognizes the values
+  label <em>NOTE:</em> will be used. Lynx recognises the values
   <em>caution</em> and <em>warning</em>, for which, respectively,
   the labels <em>CAUTION:</em> or <em>WARNING:</em> will be used.
   The NOTE element can have an ID attribute, which will be treated
@@ -2723,7 +2723,7 @@ named <em>A</em>nchor.
   "http://www.w3.org/MarkUp/html3/Contents.html">HTML 3.0</a> and
   later specifications provide for two classes of quotation in HTML
   documents. Block quotes, designated by the BLOCKQUOTE element (or
-  it's abbreviated synonym BQ in HTML 3.0), have implied paragraph
+  its abbreviated synonym BQ in HTML 3.0), have implied paragraph
   breaks preceding and following the start and end tags for the
   block. Character level quotes, designated by the Q element, in
   contrast are simply directives in the markup to insert an
@@ -2784,7 +2784,7 @@ named <em>A</em>nchor.
   make choices based on textual information. These have been termed
   <em>client-side-image-maps</em>.</p>
 
-  <p>Lynx recognizes and processes the MAP container element and
+  <p>Lynx recognises and processes the MAP container element and
   its AREA elements, and will create a menu of links for the HREF
   of each AREA when the link created for the IMG element with a
   USEMAP attribute is activated. The menu uses the ALT attributes
@@ -2822,7 +2822,7 @@ named <em>A</em>nchor.
   "#lynx.cfg">lynx.cfg</a>), and the default can be toggled via the
   <em>-ismap</em> command line switch.</p>
 
-  <p>Lynx also recognizes the <a href=
+  <p>Lynx also recognises the <a href=
   "http://www.w3.org/MarkUp/html3/Contents.html">HTML 3.0</a> FIG
   and OVERLAY elements, and will handle them as intended for text
   clients. These are the ideal way to handle
@@ -2860,7 +2860,7 @@ field is omitted, the URL defaults to th
 A <em>no-cache</em> directive is implied when the <em>Refresh</em>
 if for the same URL.
 
-  <p>Lynx recognizes and processes <em>Refresh</em> directives in
+  <p>Lynx recognises and processes <em>Refresh</em> directives in
   META elements, but puts up a labeled link, typically in the upper
   left corner of the display, indicating the number of seconds
   intended before a refresh, and the URL for the refresh, instead
@@ -2896,7 +2896,7 @@ if for the same URL.
   <p>State Management via cookie exchanges originally was
   implemented by Netscape, and such cookies are now designated as
   <em>Version 0</em>. A more elaborate format for cookies,
-  designated as <em>Version 1</em>, is being standardized by the
+  designated as <em>Version 1</em>, is being standardised by the
   IETF (Internet Engineering Task Force). Lynx supports both
   <em>Version 0</em> and <em>Version 1</em> cookie exchanges. This
   support can be disabled by default via the SET_COOKIES symbol in
@@ -2908,7 +2908,7 @@ if for the same URL.
   SET_COOKIES is TRUE, Lynx will accept all cookies. Additionally,
   the cookies that are automatically accepted or rejected by Lynx
   can be further modified with the COOKIE_ACCEPT_DOMAINS and
-  COOKIE_REJECT_DOMAINS options in your .lynxrc file, each of which
+  COOKIE_REJECT_DOMAINS options in your .etc/lynxrc file, each of which
   is a comma-separated list of domains to perform the desired
   action. The domain listed in these options must be identical to
   the domain the cookie comes from, there is no wildcard matching.
@@ -2931,7 +2931,7 @@ if for the same URL.
   Page</a>. If Lynx has been compiled with the
   --enable-persistent-cookies flag, then unexpired cookies will be
   stored between sessions in the filename set with the COOKIE_FILE
-  option in your .lynxrc.</p>
+  option in your .etc/lynxrc.</p>
 
   <p>A common use of cookies by http servers is simply to track the
   documents visited by individual users. Though this can be useful
@@ -2961,7 +2961,7 @@ if for the same URL.
   <h2 id="id-Cache"><a name="Cache" id="Cache"><em>Cached
   Documents</em></a></h2>
 
-  <p>A list of documents which are in lynx's internal cache is
+  <p>A list of documents which are in lynx' internal cache is
   accessible through hypothetical <em>Cache Jar</em> which can be
   examined via the CACHE_JAR keystroke command, normally mapped to
   <em>Ctrl-X</em>.</p>
@@ -3034,7 +3034,7 @@ if for the same URL.
   <h2 id="id-Sessions"><a name="Sessions" id=
   "Sessions"><em>Sessions</em></a></h2>
 
-  <p>Lynx's current state (all information about the user's current
+  <p>Lynx' current state (all information about the user's current
   activity with lynx) is called a session. Sessions are useful in
   particular if you are in the middle of exploring something on the
   web and you were forced to stop abruptly, losing any trace of
@@ -3129,7 +3129,7 @@ if for the same URL.
               -center:off
               -center=off
               -center-</code>
-</pre>Lynx recognizes "1", "+", "on" and "true" for true values,
+</pre>Lynx recognises "1", "+", "on" and "true" for true values,
 and "0", "-", "off" and "false" for false values. Other
 option-values are ignored.
 
@@ -3142,7 +3142,7 @@ option-values are ignored.
       Therefore runtime configuration values are not reflected in
       the help-message.</p>
 
-      <p>Capitalized items in the option summary indicate that a
+      <p>Capitalised items in the option summary indicate that a
       substitution must be made. These are the options:</p>
 
       <dl>
@@ -3184,7 +3184,7 @@ option-values are ignored.
 
         <dt><code>-assume_unrec_charset=MIMENAME</code></dt>
 
-        <dd>use this instead of unrecognized charsets.</dd>
+        <dd>use this instead of unrecognised charsets.</dd>
 
         <dt><code>-auth=ID:PW</code></dt>
 
@@ -3301,8 +3301,8 @@ option-values are ignored.
         terminal description to determine whether color mode is
         possible, and this flag is not needed and thus
         unavailable.) A saved show_color=always setting found in a
-        .lynxrc file at startup has the same effect. A saved
-        show_color=always found in .lynxrc on startup is overridden
+        .etc/lynxrc file at startup has the same effect. A saved
+        show_color=always found in .etc/lynxrc on startup is overridden
         by this flag.</dd>
 
         <dt><code>-connect_timeout=N</code></dt>
@@ -3313,7 +3313,8 @@ option-values are ignored.
         <dt><code>-cookie_file=FILENAME</code></dt>
 
         <dd>specifies a file to use to read cookies. If none is
-        specified, the default value is ~/.lynx_cookies for most
+        specified, the default value is ~/.etc/cookies.txt on
+        MirBSD, ~/.lynx_cookies for most
         systems, but ~/cookies for MS-DOS.</dd>
 
         <dt><code>-cookie_save_file=FILENAME</code></dt>
@@ -3566,7 +3567,7 @@ option-values are ignored.
 
         <dd>force color mode off, overriding terminal capabilities
         and any <em>-color</em> flags, <em>COLORTERM</em> variable,
-        and saved .lynxrc settings.</dd>
+        and saved .etc/lynxrc settings.</dd>
 
         <dt><code>-noexec</code></dt>
 
@@ -3682,7 +3683,7 @@ option-values are ignored.
 
         <dd>show source preparsed and reformatted when used with
         -source or in source view ('<em>\</em>'). May be useful for
-        debugging of broken HTML markup to visualize the difference
+        debugging of broken HTML markup to visualise the difference
         between SortaSGML and TagSoup <a href=
         "keystrokes/option_help.html#tagsoup">recovery modes</a>,
         switched by '<em>^V</em>'.</dd>
@@ -3724,7 +3725,7 @@ option-values are ignored.
         -restrictions=[option][,option][,option]...</em></dd>
 
         <dd>
-          The list of recognized options is printed if none are
+          The list of recognised options is printed if none are
           specified.
 
           <dl>
@@ -3754,7 +3755,7 @@ option-values are ignored.
 
             <dt>chdir</dt>
 
-            <dd>disallow command which changes Lynx's working
+            <dd>disallow command which changes Lynx' working
             directory.</dd>
 
             <dt>default</dt>
@@ -3881,7 +3882,7 @@ option-values are ignored.
 
             <dt>options_save</dt>
 
-            <dd>disallow saving options in .lynxrc.</dd>
+            <dd>disallow saving options in .etc/lynxrc.</dd>
 
             <dt>outside_ftp</dt>
 
@@ -3921,7 +3922,7 @@ option-values are ignored.
 
             <dt>telnet_port</dt>
 
-            <dd>disallow specifying a port in telnet G)oto's.</dd>
+            <dd>disallow specifying a port in telnet G)oto.</dd>
 
             <dt>useragent</dt>
 
@@ -4008,7 +4009,7 @@ option-values are ignored.
                     </tt>
 </pre>generates HTML source listing the files in the current
 directory. Each file is marked by an HREF relative to the parent
-directory. Add a trailing slash to make the HREF's relative to the
+directory. Add a trailing slash to make the HREFs relative to the
 current directory:
           <pre>
 <tt>
@@ -4049,7 +4050,7 @@ current directory:
 
         <dt><code>-tagsoup</code></dt>
 
-        <dd>initialize DTD with "TagSoup" tables, <a href=
+        <dd>initialise DTD with "TagSoup" tables, <a href=
         "keystrokes/option_help.html#tagsoup">more
         details</a>.</dd>
 
@@ -4118,7 +4119,7 @@ current directory:
 
             <dt>8</dt>
 
-            <dd>config (lynx.cfg and .lynxrc contents)</dd>
+            <dd>config (lynx.cfg and .etc/lynxrc contents)</dd>
 
             <dt>16</dt>
 
@@ -4228,7 +4229,7 @@ current directory:
 
   <p>Lynx has several levels of customization: from the Options
   Menu (accessible on-line, and possibly stored in your local
-  .lynxrc file), via command-line switches on startup (mainly for
+  .etc/lynxrc file), via command-line switches on startup (mainly for
   batch processing). The most important and numerous default
   settings are stored in the Lynx configuration file
   <em>lynx.cfg</em>.</p>
